java/src/main/java/com/adhoc/flight/utils/QueryUtils.java

```diff
@@ -0,0 +1,241 @@
+            print(Prefixes.INFORMATION, "Running query.");
+        }
+
+        /**
+         * Prints the query results to the console.
+         * 
+         * @param vectorSchemaRoot the vector schema root from which to get
+         *        the results.
+         */
+        public static void printResults(VectorSchemaRoot vectorSchemaRoot) {
+
+            print(Fillers.HEADER, "Query results");
+
+            /*
+             * This prints out the results, each value in its corresponding
+             * row and column, with different columns separated by tabs and
+             * properly aligned with their field names.
+             *
+             * If you intend to read from a binary file containing the data of
+             * the VectorSchemaRoot with the query results, the following code
+             * snippet should help you out:
+             * 
+             * ============================ CODE =============================
+             * 
+             * public static void printFromBinaryFile(File file) {
+             *
+             *     try (ArrowStreamReader reader = new ArrowStreamReader(
+             *          new FileOutputStream(file), new RootAllocator(Long.MAX_VALUE))) {
+             *
+             *         // Instantiating a new VectorSchemaRoot based on the binary info.
+             *         VectorSchemaRoot batchRead = reader.getVectorSchemaRoot;
+             *         
+             *         // Updating the data inside the VectorSchemaRoot.
+             *         reader.loadNextBatch();
+             *         
+             *         // Printing out the results.
+             *         System.out.println(batchRead.contentToTSVString());
+             *     }
+             * }
+             * 
+             * ============================ INFO =============================
+             * 
+             * For more information on this, please refer to the documentation:
+             *     <https://arrow.apache.org/docs/java/ipc.html>
+             */
+            System.out.println(vectorSchemaRoot.contentToTSVString());
+
+            print(Fillers.FOOTER, "Number of records retrieved: " + vectorSchemaRoot.getRowCount());
+        }
+
+        /**
+         * Prints the provided exception to the console.
+         * 
+         * @param e the exception to be printed out.
+         */
+        public static void printExceptionOnClosed(Exception e) {
+            print(Prefixes.ERROR, e.getMessage());
+            e.printStackTrace();
+        }
+
+        private static void print(Prefixes prefixes, String message) {
+            System.out.println(prefixes.toFormattedString() + " " + message);
+        }
+        
+        private static void print(Fillers fillers, String message) {
+        	String filler = fillers.toFormattedString();
+        	System.out.println(filler + " " + message + " " + filler);
+        }
+    }
+}
+
+enum Fillers {
+    HEADER('-'),
```

Comment:
This seems overly complicated? Can we juts have two String constants ---------------- and =========== (ass 18 characters)...?

Reply:
I agree, I over-engineered this. Solved.